client/components/Common/Icon.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import styled from 'styled-components'
+import { storiesOf } from '@storybook/react'
+import Icon from './Icon'
+
+storiesOf('Common/Icon', module)
+  .add('Default', () => <Icon name="help-circle" spin={false} />)
```

Comment:
nits:
As a default example, I think it should be described **only** required props.
```suggestion
  .add('Default', () => <Icon name="help-circle" />)
```